test/test_quantized_tensor.py

```diff
@@ -112,9 +112,8 @@ def test_qtensor_creation(self):
         self.assertEqual(q.q_zero_point(), q_el.q_zero_point())
         self.assertEqual(q.dtype, q_el.dtype)
 
-        # create via empty_like but change the dtype (currently not supported)
-        with self.assertRaises(RuntimeError):
-            torch.empty_like(q, dtype=torch.qint8)
+        # create via empty_like but change the dtype
+        self.assertEqual(torch.qint8, torch.empty_like(q, dtype=torch.qint8).dtype)
```

Comment:
err..how does this work?  Is the zero_point set correctly?

Reply:
yeah. Updated test to add this check.